src/view/HomeView.java

```diff
@@ -0,0 +1,111 @@
+package view;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+import interface_adapter.ViewManagerModel;
+
+/**
+ * The home page view for the application.
+ */
+public class HomeView extends JPanel implements PropertyChangeListener {
+
+    public final String viewName = "home view";
+    private ViewManagerModel viewManagerModel;
+
+    public HomeView(ViewManagerModel viewManagerModel) {
+        this.viewManagerModel = viewManagerModel;
+
+        // Create buttons for different use cases
+        JButton IDButton = new JButton("Search by ID");
+        JButton CuisineButton = new JButton("Search by Cuisine");
+        JButton NameButton = new JButton("Search by Name");
+        JButton RecipeButton = new JButton("Display User Recipes Collection");
+
+        // will be removed in the future, just for testing purposes
+        JButton button4 = new JButton("Display Favourite Recipes");
```

Comment:
I can remove these now that RecipeView brings me to those views, I had these 2 buttons in so that I could access DisplayFavouriteView and DisplayTaggedView/DisplayTagsView without going through RecipeView which wasn't finished at the time.

Reply:
They have been removed in the latest commit.